src/basics/Problem42.java

```diff
@@ -0,0 +1,20 @@
+package basics;
+
+import java.util.Scanner;
+import java.io.Console;
+
+public class Problem42 {
+  
+  public static void main(String []args) {
+    //creating a reference to the Console class
+    Console con = System.console();
+
+    System.out.print("Enter your password: ");
+    //saving the input as a Char in char array
+    char[] password = con.readPassword();
+    
+    //converting char array to String
```

Comment:
Use present form when commenting, i.e.
```java
// Convert char array to string
```
Comment should start with capital letter, and there should be a space after `//`